node/index.js

```diff
@@ -98,7 +98,9 @@ module.exports = options => {
 		next();
 	});
 
-	let hasher = str => str;
+	let hasher = {
```

Comment:
wasn't sure about this and where it's used, but I assume it should have the same API as the actual hasher...